Source/FileWatcherEx/Helpers/SymlinkAwareFileWatcher.cs

```diff
@@ -178,7 +180,7 @@ internal void TryRegisterFileWatcherForSymbolicLinkDir(string path)
     {
         try
         {
-            if (IsSymbolicLinkDirectory(path) && IncludeSubdirectories && !FileWatchers.ContainsKey(path))
+            if (IsSymbolicLinkDirectory(path) && !FileWatchers.ContainsKey(path))
```

Comment:
@PhantomGamers removing this condition cause the test failed

![image](https://github.com/d2phap/FileWatcherEx/assets/3154213/f07d32a0-0e68-451f-8613-f7c5b2f34137)

Reply:
My bad I didn't actually check the tests, just checked to see if the code fixed the issue I was having. I'll look into it right now